Review backup and restore assemblies for CQA (#4846)
theforeman/foreman-documentation: guides/common/modules/proc_restoring-from-a-full-backup.adoc

= Restoring from a full backup

[role="_abstract"]

> Restore {ProjectName} or {SmartProxyServer} from a full backup return the system to the state at the time of the backup.

Reviewer: ```suggestion
Restore {ProjectServer} or {SmartProxyServer} from a full backup return the system to the state at the time of the backup.
```

Unsure about "system" here. If there's no other comment, we should just keep it.

Author: I think this reference to "system" is a generic one and thus safe. I'm leaving the thread open in case anyone disagrees.

Reviewer: OK, agreed. Then maybe just the suggestion ProjectName -> ProjectServer

When the restore process completes, all processes are online, and all databases and system configuration revert to the state at the time of the backup.

.Prerequisites

> * The {ProjectName} instance must have the same host name, configuration, and be the same minor version (X.Y) as the original system.

Reviewer: ```suggestion
* Your {ProjectServer} must have the same host name, configuration, and be the same minor version (X.Y) as the original system.
```

* The target directory must exist and be writable.
The target directory is read from the configuration files contained within the backup archive.
ifndef::foreman-deb[]

> * If the backed up system had FIPS enabled, the system on which you are restoring must also be FIPS enabled.

Reviewer: ```suggestion
* If the backed up system had FIPS enabled, the system on which you are restoring must also have FIPS enabled.
```

+
The restore process can take a long time to complete, because of the amount of data to copy.

.Next steps

> * If you create a new instance of the {ProjectServer} or {SmartProxyServer}, decommission the old instance after restoring the backup.

Reviewer: ```suggestion
* If you create a new instance of {ProjectServer} or {SmartProxyServer}, decommission the old instance after restoring the backup.
```
